ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -91,13 +90,18 @@ message GetParticipantIdResponse {
   string participant_id = 1;
 }
 
-// Required authorization: ``HasRight(ParticipantAdmin)``
+// Required authorization: ``HasRight(ParticipantAdmin) OR HasRight(IdentityProviderAdmin)``
 message GetPartiesRequest {
 
   // The stable, unique identifier of the Daml parties.
   // Must be valid PartyIdStrings (as described in ``value.proto``).
   // Required
   repeated string parties = 1;
+
+  // The id of the ``Identity Provider``
```

Comment:
```suggestion
  // The id of the ``Identity Provider`` whose parties should be retrieved.
```

Reply:
apply the analogous change to the other requests and responses being modified here.

The goal is that a reader understand how the `identity_provider` field relates to the other data in the request and response data.